library/src/main/java/com/getstream/sdk/chat/rest/core/Client.java

```diff
@@ -589,6 +595,19 @@ private synchronized void connect() {
         });
     }
 
+    private synchronized void anonymousConnect() {
```

Comment:
we don't need a separate connect method; we already have `anonymousConnection`. we can use that to set things up